src/occupational_classification/utils/constants.py

```diff
@@ -0,0 +1,6 @@
+"""Module for common constant definitions.
+
+This module contains constants used across the occupational classification utilities.
+"""
+
+SOC_CODE_LENGTH = 5
```

Comment:
What does this do? Also for SOC the expected code length is 4 (for Unit Groups).